plugin-test/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+    The MIT License (MIT)
+    
+    ${PROJECT_NAME} Copyright (c) ${YEAR} ${USER}
```

Comment:
I have no problem with removing the license headers it is just a habit i get from university